guides/source/association_basics.md

```diff
@@ -1961,9 +1961,13 @@ To use delegated types, we need to model our data as follows:
 This eliminates the need to define attributes in a single table that are
 unintentionally shared among all subclasses.
 
+NOTE: The delegated types examples below are independent of the STI “Vehicle” example in the previous section.
+These examples start from scratch with an `Entry` model(the delegator) and two concrete types: `Message` and `Comment`.
+
 ### Generating Models
 
-In order to apply this to our example above, we need to regenerate our models.
```

Comment:
Simply removing this line should work, I think.

Reply:
@shivabhusal – thank you, I agree. It may be more clean and explicit. It is updated and looks like below:
<img width="1371" height="662" alt="image" src="https://github.com/user-attachments/assets/f867bf62-b743-4f6e-b5ee-7946616bb9cb" />